src/documents/SettingsStore.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Settings Store
+layout: default
+category: Infusion
+---
+
+The Infusion [Preferences Framework](PreferencesFramework.md) uses a _Settings Store_ to read and
+write preferences.
+
+## fluid.prefs.store ##
+
+A base grade to be used by any preference setting store.
+This component uses the following base [grades](ComponentGrades.md):
+
+* [`fluid.prefs.dataSource`](#fluid-prefs-datasource)
```

Comment:
It seems the link for this is "#fluidprefsdatasource" instead of "#fluid-prefs-datasource".